CHANGELOG.md

```diff
@@ -4,14 +4,18 @@ All notable changes to this project will be documented in this file.
 
 ## [4.6.0] - TBD
 
-Wazuh commit: TBD \
-Release report: TBD
-
```

Comment:
Shouldn't this be maintained?

Reply:
Done: https://github.com/wazuh/wazuh-qa/pull/4348/commits/6bc115abf08895aa5419cfd72d52972d05246d60